bokeh/plotting/_docstring.py

```diff
@@ -93,8 +93,9 @@ def _docstring_extra(extra_docs):
     return "" if extra_docs is None else extra_docs
 
 def _docstring_header(glyphclass):
-    module = "markers" if issubclass(glyphclass, Marker) else "glyphs"
-    return f"Configure and add :class:`~bokeh.models.{module}.{glyphclass.__name__}` glyphs to this Figure."
+    glyph_class = "Scatter" if issubclass(glyphclass, Marker) else glyphclass.__name__
+    # glyph_class = "Scatter" if issubclass(glyphclass, Marker) else str(glyphclass.__name__)
```

Comment:
should delete commented code

Reply:
Absolutley, I guess I hit the 'commit' button a bit too ealry on this one. Should be fixed with my next commit! 😀